README.md

```diff
@@ -54,6 +54,85 @@ test.all(common)
 
 A valid (read: that follows this interface) IPFS core implementation, must expose the following API.
 
+## Group Files
+
+### `Add`
+
+> Store a Unixfs file or directory of files .
+
+##### `Go` **WIP**
+
+##### `JavaScript` - ipfs.add(path, [options, callback])
+
+`path` The path to a file to be added to IPFS. Can be of type:
```

Comment:
even `files` is kind of misleading, since a stream !== a file. heck, maybe even something as nebulous as `data`. the real problem here is that this api call is becoming a kitchen sink; it becomes hard to define concrete terminology when you accept so many different things in one place.